src/helper/queryStrHelper.js

```diff
@@ -1,127 +0,0 @@
-const makeGetRentalQueryStr = (baseQueryStr, queryParams) => {
-	const { customerId, gameId, status, startDate, endDate } = queryParams
-
-	let queryStr = baseQueryStr
-	const queryArgs = []
-
-	if (Object.keys(queryParams).length !== 0) queryStr += ' WHERE '
-
-	let customerStr = '1 = 1'
-	if (customerId) {
```

Comment:
Qual foi o proposito de inicializar a variável desta forma? 🤔 
Se for apenas para inicializar ela como uma string, poderia simplesmente declarar como `""`.